cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -199,6 +201,36 @@ func generateInitramfsMounts() (err error) {
 	return nil
 }
 
+func canInstallAndRun(mst *initramfsMountsState) (bool, error) {
+	currentSeed, err := seed.Open(boot.InitramfsUbuntuSeedDir, mst.recoverySystem)
+	if err != nil {
+		return false, err
+	}
+	preseedSeed, ok := currentSeed.(seed.PreseedCapable)
+	if !ok {
+		return false, nil
+	}
+
+	if !preseedSeed.HasArtifact("preseed.tgz") {
```

Comment:
let's have a TODO here about relaxing later this condition

Reply:
Done in c5a44c0c1b98e1301786e6567898951bbbeebdfc